src/components/ListContainer/ListContainer.tsx

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from '@material-ui/core';
+import React, { FC } from 'react';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+    maxWidth: '853px',
```

Comment:
This seems really arbitrary. What about one of the `theme.breakpoints.values.{x}`? (theme is the arg passed into this function)

Reply:
Good call; I totally didn't think about using the theme. I just did the math from the Figma design 🙃